main/src/cgeo/geocaching/InstallWizardActivity.java

```diff
@@ -453,6 +457,18 @@ private void requestGpxfolder() {
         }
     }
 
+    private static boolean broutertilesFolderNeedsMigration() {
+        return Settings.isBrouterAutoTileDownloads() && PersistableFolder.BROUTER_TILES.isLegacy();
```

Comment:
To be on the safeside, we should also check if BRouter is installed at all, shouldn't we?

Reply:
The option to activate auto downloading is greyed out when BRouter is not installed, so you can activate it only by importing a config file or via Settings => System => View Settings.

But anyway, better to add a test here.